gallery/visualtests/nxDrawer.js

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+const { setupBrowser } = require('./testUtils');
+
+describe('NxDrawer', function() {
+  const viewportSize = { width: 1366, height: 1000 };
+
+  const openDrawerFromPage = ({ waitAndGetElements, wait }) => async (buttonId, drawerId) => {
+    const [openDrawerButton] = await waitAndGetElements(`#${buttonId}`);
+
+    await openDrawerButton.click();
+
+    await waitAndGetElements(`#${drawerId}`);
+
+    // Wait for animation.
+    await wait(200);
+  };
+
+  describe('NxDrawer with Title, Footer, and non-overflowing Content', function() {
+    const browserSetup = setupBrowser('#/NxDrawerExample', false);
+    const { getPage, checkFullPageScreenshot, a11yTest } = browserSetup;
+    const openDrawer = openDrawerFromPage(browserSetup);
+
+    const buttonId = 'nx-drawer-with-footer-open-button';
+    const drawerId = 'nx-drawer-with-footer';
+
+    it('looks right', async function() {
+      await getPage().setViewport(viewportSize);
+      await openDrawer(buttonId, drawerId);
+      await checkFullPageScreenshot();
+    });
+
+    it('passes a11y', async function() {
+      await getPage().setViewport(viewportSize);
+      await openDrawer(buttonId, drawerId);
+      await a11yTest();
```

Comment:
```suggestion
      await a11yTest()();
```

Reply:
And similar below

Reply:
> And similar below

Actually you probably also need to pass parameters into `a11yTest` to tell it to check the entire page rather than searching for `.gallery-example-live`.  See `nxPageLayout.js` for what I mean.

Reply:
> > And similar below
> 
> Actually you probably also need to pass parameters into `a11yTest` to tell it to check the entire page rather than searching for `.gallery-example-live`. See `nxPageLayout.js` for what I mean.

Fixed: 9c21e07f43dcbb21c13cb39d30127aadff0b9b4f